docs/performance.rst

```diff
@@ -62,6 +62,11 @@ To determine if PowerDNS is unable to keep up with packets, determine
 the value of the :ref:`stat-qsize-q` variable. This represents the number of
 packets waiting for database attention. During normal operations the
 queue should be small.
+This number is a total over all receiver threads.
+
+The :ref:`setting-max-queue-length` and :ref:`setting-overload-queue-length` settings determine how PowerDNS deals with growing queues.
+If the queue for a single receiver thread (and its associated distributor threads) grows beyond the ``overload`` number, queries are answered only from the packet cache so the database can hopefully recover.
```

Comment:
When reading this, an immediate question popped up in my mind: what happens to packet cache misses in the overload case? Arte they dropped or maybe answered with ServFail???

Reply:
the settings page (linked) answers that: they are dropped

Reply:
OK, that's fine.